onset_maproom/layout.py

```diff
@@ -22,19 +29,67 @@
 LIGHT_GRAY = "#eeeeee"
 
 
+def adm_borders(shapes):
+    with psycopg2.connect(**CONFIG["db"]) as conn:
+        s = sql.Composed(
+            [
+                sql.SQL("with g as ("),
+                sql.SQL(shapes),
+                sql.SQL(
+                    """
+                    )
+                    select
+                        g.label, g.key, g.the_geom
+                    from g
+                    """
+                ),
+            ]
+        )
+        df = pd.read_sql(s, conn)
+
+    df["the_geom"] = df["the_geom"].apply(lambda x: wkb.loads(x.tobytes()))
+    df["the_geom"] = df["the_geom"].apply(
+        lambda x: x if isinstance(x, MultiPolygon) else MultiPolygon([x])
+    )
+    shapes = df["the_geom"].apply(shapely.geometry.mapping)
+    for i in df.index: #this adds the district layer as a label in the dict
+        shapes[i]['label'] = df['label'][i]
+    return {"features": shapes}
+
+
+def make_adm_overlay(adm_list, adm_lev, adm_weight):
+    border_id = f'borders_adm{adm_lev}'
+    adm_check = False
+    if adm_lev == 1:
+        adm_check = True
```

Comment:
I suggest making `is_checked` a parameter of this function so it can be used with different default layers in different contexts.